src/components/MapView/utils.ts

```diff
@@ -0,0 +1,13 @@
+function getBounds(waypoints: Array<[number, number]>): {southWest: [number, number]; northEast: [number, number]} {
+    const lngs = waypoints.map((waypoint) => waypoint[0]);
+    const lats = waypoints.map((waypoint) => waypoint[1]);
```

Comment:
Hi ✋ Coming from https://github.com/Expensify/App/issues/27175

To enhance the bounds, we also should be considering the direction coordinates so it can be zoomed out from the furthest distance.